### PR TITLE
fix for userconfig PR 2

### PR DESCRIPTION
**Pull request by Bilb**

### CODE REVIEW

**Review by yougotwill — CHANGES_REQUESTED** [review]
We probably need to add the avatar icon that iOS and Android use as a fallback for when the pubkey is undefined when generating the closegroup avatar.

**Review by yougotwill — CHANGES_REQUESTED** [review]

File: ts/components/avatar/AvatarPlaceHolder/AvatarPlaceHolder.tsx [hunk 1]
```diff
@@ -79,20 +83,15 @@ export const AvatarPlaceHolder = (props: Props) => {
...
-            shapeRendering="geometricPrecision"
-            stroke={'var(--avatar-border-color)'}
-            strokeWidth="1"
-          />
-        </g>
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 26 26">
+        <circle fill="var(--primary-color)" cx="13" cy="13" r="13" />
+        <path
+          fill="#FFF"
+          d="M18.9 19.1c-1.5-.9-3-.8-3-.8h-3.6c-2.3 0-3.3 0-4.2.3-.9.3-1.8.8-2.8 2-.5.7-.8 1.4-1 2C6.6 24.7 9.7 26 13 26c3.3 0 6.4-1.3 8.7-3.3-.5-1.6-1.5-2.8-2.8-3.6z"
```
[the remark this answers is not in the document]
Comment (Bilb) [reply ts/components/avatar/AvatarPlaceHolder/AvatarPlaceHolder.tsx:92]: we can't pass color currently to that Icons.tsx file, so we can't move that one there.

[the remark this answers is not in the document]
Comment (Bilb) [reply ts/components/avatar/AvatarPlaceHolder/AvatarPlaceHolder.tsx:92]: those work for single color svgs
https://github.com/oxen-io/session-desktop/blob/ff4366002b7059a327eccae5b875ca97bd804bab/ts/components/icon/Icons.tsx

[the remark this answers is not in the document]
Comment (yougotwill) [reply ts/components/avatar/AvatarPlaceHolder/AvatarPlaceHolder.tsx:92]: You don't need to pass the color. You could just pass a variable prop for `path.d` and `viewbox` like in my link from the Icon type? But I guess if you want to keep it separate that is fine. I didn't know that Array was for single colors only. Will keep it in mind,

**Pull request merged**